Launcher/Classes/LocalizationHandler.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+
+class LocalizationHandler {
+    
+    let applicationStateHandler = ApplicationStateHandler()
+    let fileManager = FileManager.default
+    var jsonFilePaths: [String] = []
+    var localizedStringsFilePaths: [String] = []
+    
+    init() {
+        jsonFilePaths = allFiles(withFileExtension: "json")
+        localizedStringsFilePaths = allFiles(withFileExtension: "strings")
+    }
+    
+    func keys(for string: String) -> [String] {
+        guard let value = parseResponse(string) else { return [] }
+        
+        let resultForLocalizedStrings = keysForLocalizedStrings(value: value)
+        
+        if !resultForLocalizedStrings.isEmpty {
+            return resultForLocalizedStrings
+        } else {
+            return keysForJson(value: value)
+        }
+    }
+    
+    func parseResponse(_ response: String) -> String? {
```

Comment:
Can we add at least some header documentation (with an example) of what this does?

Reply:
Also this does not technically have to live in this class.